components/TopBar/TopBar.tsx

```diff
@@ -33,28 +49,78 @@ export const TopBar: FC<{ className?: string | undefined }> = (props) => {
   }
 
   return (
-    <div
-      className={`flex items-center text-xl dark:text-gray-300 ${className}`}
-    >
-      <FaSlackHash className="mr-2 text-gray-600 dark:text-gray-300" />
-      <span className="text-gray-900 dark:text-gray-100 uppercase">
-        {category.split('-').join(' ')}
-      </span>
-      <button
-          data-tooltip-id="info-tooltip"
-          data-tooltip-content="Description"
-          data-tooltip-place="bottom"
+    <>
+      {regEx.test(category) ? (
+        <div
+          className={`flex items-center text-xl dark:text-gray-300 ${className}`}
+        >
+          <FaSlackHash className="mr-2 text-gray-600 dark:text-gray-300" />
+          <span className="flex uppercase text-gray-900 dark:text-gray-100">
+            {isSearchFound
+              ? `Search: Results Found`
```

Comment:
when the results are found, can you just write here whatever the search query is? lets say user is searching for **web** so it should show **web** instead of this.